alicloud/resource_alicloud_cdn_domain.go

```diff
@@ -716,6 +770,64 @@ func authConfigUpdate(client *connectivity.AliyunClient, d *schema.ResourceData)
 	return nil
 }
 
+func httpsConfigUpdate(client *connectivity.AliyunClient, d *schema.ResourceData) error {
+	ov, nv := d.GetChange("https_config")
+	oldConfig, newConfig := ov.(*schema.Set), nv.(*schema.Set)
+	args := cdn.CertificateRequest{
+		DomainName: d.Id(),
+	}
+
+	if newConfig == nil || newConfig.Len() == 0 {
+		args.ServerCertificateStatus = "off"
+		_, err := client.WithCdnClient(func(cdnClient *cdn.CdnClient) (interface{}, error) {
+			return cdnClient.SetDomainServerCertificate(args)
+		})
+		if err != nil {
+			return err
+		}
```

Comment:
Line 788 needs set partial after ser domain config.

Reply:
I used `authConfigUpdate` as a template, from here: https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/alicloud/resource_alicloud_cdn_domain.go#L663. There is a `d.SetPartial("https_config")` on line 792. Why does one also need to be on line 788? It seems like in `authConfigUpdate` there is only 1 `SetPartial`. Sorry if this is a dumb question, I don't fully understand what `SetPartial` does.

Reply:
When one config has been changed successfully by invoking API, SetPartial will write down it to state even if the next change failed.